.github/workflows/production.yml

```diff
@@ -33,8 +33,8 @@ jobs:
       - name: Provision to production
         run: |
           echo Login
-          oc login "${{ secrets.OPENSHIFT_CLUSTER }}" --token="${{ secrets.OC4_PROD_TOKEN }}"
-          PROJ_TARGET=e1e498-prod bash openshift/scripts/oc_deploy_to_production.sh ${SUFFIX} apply
+          oc login "${{ secrets.OPENSHIFT_CLUSTER }}" --token="${{ secrets.OC4_TEST_TOKEN }}"
```

Comment:
`OC4_TEST_TOKEN` token should be scoped to `e1e498-test` namespace only, so nothing should be accidentally deployed to prod.